frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -12,7 +12,7 @@
           <v-icon size="large">mdi-history</v-icon>
           <strong>Payment Records</strong>
         </v-tab>
-        <v-tab v-if="hasPermission(PERMISSIONS.VIEW_FUNDING_AMOUNTS)" value="funding-allocation">
+        <v-tab v-if="hasPermission(PERMISSIONS.VIEW_FUNDING_AGREEMENT)" value="funding-allocation">
```

Comment:
Can we add a TODO to confirm permissions? Same below.

Reply:
I've added TODO